codegen/src/test/resources/q10/q10.json

```diff
@@ -0,0 +1,181 @@
+{
+    "join_structure": [
+      {
+        "primary": "Orders",
+        "foreign": "Lineitem"
+      },
+      {
+        "primary": "Customer",
+        "foreign": "Orders"
+      },
+      {
+          "primary":"Nation",
+          "foreign":"Customer"
+      }
+    ],
+    "RelationProcessFunction": [
+      {
+        "name": "Q10lineitem",
+        "relation": "lineitem",
+        "this_key": [
+          "orderkey" 
+        ],
+        "next_key": [
+          "custkey", 
+          "c_name",
+          "c_acctbal",
+          "c_phone",
+          "c_address",
+          "c_comment",
+          "n_name"
+        ],
+        "child_nodes": 3,
+        "is_Root": true,
+        "is_Last": true,
+        "rename_attribute": null,
+        "select_conditions": {
+          "operator": "&&",
+          "values": [
+            {
+              "left_field": {
+                "type": "attribute",
+                "relation": "lineitem",
+                "name": "returnflag"
+              },
+              "right_field": {
+                "type": "constant",
+                "var_type": "varchar",
+                "value": "R"
+              },
+              "operator": "=="
+            }
+          ]
+        }
+      },
+      {
+        "name": "Q10Orders",
+        "relation": "orders",
+        "this_key": [
+          "custkey"
+        ],
+        "next_key": [
+          "orderkey"
+        ],
+        "child_nodes": 2,
```

Comment:
It only has one child node.

Reply:
revised in `https://github.com/DanishAlsayed/gui-codegen/pull/26/commits/92f88668c395efd0e64745a9f0a0b3bccbb345fa`